src/plugins/ncr/sagas/updateNodeLabelsFlow.js

```diff
@@ -0,0 +1,39 @@
+import { call } from 'redux-saga/effects';
+import startCase from 'lodash/startCase';
+import changeNodeFlow from './changeNodeFlow';
+
+export default function* ({ config, pbj }) {
+  const { expectedEvent } = config;
+
+  yield call(changeNodeFlow, {
+    expectedEvent,
+    failureMessage: `Failed. The ${startCase(pbj.get('node_ref').getLabel())} failed to update.`,
+    getNodeRequestSchema: config.schemas.getNodeRequest,
+    pbj,
+    successMessage: `Success! The ${startCase(pbj.get('node_ref').getLabel())} was updated.`,
+    verify: (response) => {
+      if (!response.pbj.has('node')) {
+        return false;
+      }
+      const responseLabels = response.pbj.get('node').get('labels', []).sort();
+
+      // check if all add_labels was added
```

Comment:
@rsumilang might be easier to use isInSet on the node... iterate through add/remove is do !isInSet or isInSet, no extraction/sorting/etc. needed

Reply:
@gdbrown not sure how thats simpler. it seems i have to loop through the results either way and check each item?